### PR TITLE
Interface setup

### PR DESCRIPTION
**Pull request by Logicon211**

### CODE REVIEW

**Comment by Logicon211** [comment]
The commit list is pretty huge but we did a quick test to make sure. We can squash and commit this into master when we're ready. 

So **please** don't forget to squash

File: README.md [hunk 1]
```diff
@@ -12,15 +12,15 @@ python stix_shifter.py `<data source>` `<input format>` `<STIX pattern>`
...
+`python3 stix_shifter.py "qradar" "query" "[ipv4-addr:value = '198.51.100.1' or ipv4-addr:value = '198.51.200.1']"`
 
 Returns:
 
 `SELECT * FROM events WHERE destinationip='192.168.122.84' OR sourceip='192.168.122.84' OR destinationip='192.168.122.83' OR sourceip='192.168.122.83'`
 
 Example of converting AQL events to STIX:
 
-`python3 stix_shifter.py "qradar" "qradar_events" '[{"starttime": 1524227777191, "protocolid": 255, "sourceip": "9.21.123.112", "logsourceid":126, "qid": 55500004, "sourceport": 0, "eventcount": 1, "magnitude": 4, "identityip": "0.0.0.0", "destinationip": "9.21.123.112", "destinationport": 0, "category": 10009, "username": null}]'`
+`python3 stix_shifter.py "qradar" "results" '[{"starttime": 1524227777191, "protocolid": 255, "sourceip": "9.21.123.112", "logsourceid":126, "qid": 55500004, "sourceport": 0, "eventcount": 1, "magnitude": 4, "identityip": "0.0.0.0", "destinationip": "9.21.123.112", "destinationport": 0, "category": 10009, "username": null}]'`
```
Comment (mdazam1942) [inline README.md]: you already fixed it locally but just to keep track. `python3` ->  `python`

File: src/json_to_stix/transformers.py [hunk 2]
```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/env python3
```
Comment (mdazam1942) [inline src/json_to_stix/transformers.py]: as discussed we don't need this header

File: src/modules/qradar/qradar_module.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python3
```
Comment (mdazam1942) [inline src/modules/qradar/qradar_module.py]: we don't need this header as well i think

File: src/modules/qradar/qradar_translator.py [hunk 4]
```diff
@@ -1,39 +1,13 @@
-# Module to handle QRadar conversion
-import logging
+#!/usr/bin/env python3
```
Comment (mdazam1942) [inline src/modules/qradar/qradar_translator.py]: we can remove this as well if we dont need it

**Pull request merged**